src/cargo-fmt/main.rs

```diff
@@ -122,8 +122,26 @@ fn handle_command_status(status: Result<i32, io::Error>, opts: &getopts::Options
     }
 }
 
-fn get_version(verbosity: Verbosity) -> Result<i32, io::Error> {
-    run_rustfmt(&BTreeSet::new(), &[String::from("--version")], verbosity)
+fn get_version() -> Result<i32, io::Error> {
+    let mut status = vec![];
```

Comment:
Since we only run `rustfmt` once, we need not use a vector to keep an exit status.

Reply:
Thanks review. I avoided a vector.